worldengine/cli/main.py

```diff
@@ -186,32 +183,18 @@ def __seems_protobuf_worldfile__(world_filename):
     return worldengine_tag == World.worldengine_tag()
 
 
-def __seems_pickle_file__(world_filename):
-    last_byte = __get_last_byte__(world_filename)
-    return str(last_byte) == '.'
-
-
 def load_world(world_filename):
     pb = __seems_protobuf_worldfile__(world_filename)
-    pi = __seems_pickle_file__(world_filename)
-    if pb and pi:
-        print("we cannot distinguish if the file is a pickle or a protobuf "
-              "world file. Trying to load first as protobuf then as pickle "
-              "file")
+    if pb:
+        # print("we cannot distinguish if the file is a pickle or a protobuf "
+        #       "world file. Trying to load first as protobuf then as pickle "
+        #       "file")
```

Comment:
Another part of code where I wasn't sure whether I should permanently remove it. Maybe other formats will have to be checked for in the future within this function and it might be a useful little reminder? Maybe it's completely useless, though.